reporter/src/main/kotlin/reporters/StaticHtmlReporter.kt

```diff
@@ -22,10 +22,12 @@ package com.here.ort.reporter.reporters
 import ch.frankel.slf4k.*
 
 import com.here.ort.model.VcsInfo
+import com.here.ort.model.config.ScopeExclude
 import com.here.ort.utils.isValidUrl
 import com.here.ort.utils.log
 
 import java.io.File
+import java.util.*
```

Comment:
Nit: We should not use a wildcard import here.

Reply:
Done.